content/W2017/W17-Office-Hours.md

```diff
@@ -5,29 +5,44 @@ Tags: office
 Author: Erin Edward
 Summary: Announcing our Office Hours for Winter 2017
 
-Monday:
+Office Location:
```

Comment:
if you want separate lines, add an extra new line between

otherwise you can put this both on the same line